source/guides/integrate-and-extend.rst

```diff
@@ -0,0 +1,40 @@
+Integrate and extend Mattermost
+================================
+
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+  :titlesonly:
+
+    AI Copilot plugin </integrate/ai-copilot-plugin>
+    Channel Export plugin </integrate/export-mattermost-channel-data>
+    GitHub plugin </integrate/github-interoperability>
+    GitLab plugin </integrate/gitlab-interoperability>
+    Jira </integrate/jira-interoperability>
+    Legal hold
+    Performance metrics
+    Microsoft Teams </integrate/ms-teams-interoperability>
+    ServiceNow </integrate/servicenow-interoperability>
+    Zoom </integrate/zoom-interoperability>
+
+Mattermost offers many ways to add product functionality, custom-tailor your workflows, and connect Mattermost to your mission-critical tools. 
+
+Interoperability with pre-packaged integrations
+-----------------------------------------------
+
+Your Mattermost deployment comes with the following integrations you can configure and use:
+
+* :doc:`AI Copilot plugin </integrate/ai-copilot-plugin>` - Connect and use the Mattermost AI Copilot integration.
+* :doc:`Channel Export plugin </integrate/export-mattermost-channel-data>` - Learn how to export channel data.
+* :doc:`GitHub interoperability </integrate/github-interoperability>` - Connect your GitHub instance to your Mattermost instance.
+* :doc:`GitLab interoperability </integrate/gitlab-interoperability>` - Connect your GitLab instance to your Mattermost instance.
+* :doc:`Jira interoperability </integrate/jira-interoperability>` - Connect your Jira instance to your Mattermost instance.
+* :doc:`Legal hold
+* :doc:`Performance metrics
```

Comment:
I wonder if we should move the Metrics plugin out of this category. It's not an integration, but more of a tool. It's more like "Calls" and less like the "GitHub" plugin.

Reply:
That's a really good point, @hanzei, and I agree with your classification of the Metrics plugin. Legal hold and channel export would also fall into this category. Perhaps the way forward is to include only the interoperability plugins here. Tool-based integrations would be documented as any other product feature. Will think on this more. Really appreciate your input here.

Reply:
Features built as plugins are documented like features; 3rd party interoperability integrations are collected together, in the docs, under Integrate & Extend from the landing page.

I agree that we'll want to update the plugin metadata so the Marketplace entries point to the docs pages -- both the in-product Marketplace as well as the external one via mm.com/marketplace. These tasks will be completed outside of this PR.